sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/SqlScriptingParserSuite.scala

```diff
@@ -205,13 +206,15 @@ class SqlScriptingParserSuite extends SparkFunSuite with SQLHelper {
         |  SELECT a, b, c FROM T;
         |  SELECT * FROM T;
         |END lbl_end""".stripMargin
-
+    val exception = intercept[SqlScriptingException] {
+      parseScript(sqlScriptText)
+    }
     checkError(
-      exception = intercept[SqlScriptingException] {
-        parseScript(sqlScriptText)
-      },
+      exception = exception,
       errorClass = "LABELS_MISMATCH",
```

Comment:
The `errorClass` doesn't exist anymore. Please, rebase your PR on the recent master.

Reply:
@MaxGekk rebased